opencti-platform/opencti-front/lang/front/fr.json

```diff
@@ -657,7 +657,7 @@
   "Do you want to delete this country?": "Souhaitez-vous supprimer ce pays ?",
   "Do you want to delete this course of action?": "Souhaitez-vous supprimer cette conduite à suivre ?",
   "Do you want to delete this CSV ingester?": "Voulez-vous supprimer cet ingéreur CSV ?",
-  "Do you want to delete this CSV mapper ?": "Voulez-vous supprimer ce mappage CSV ?",
+  "Do you want to delete this CSV mapper?": "Voulez-vous supprimer ce mappage CSV ?",
```

Comment:
**not related** to the PR, but we would need at some point to review the auto translations. In french, CSV mapper has currently 3 different translations "mappage CSV", "mappeur CSV", "mapping CSV"